stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/common/extensions/internal/Context.kt

```diff
@@ -54,11 +58,31 @@ internal fun Context.copyToClipboard(text: String) {
     clipboard.setPrimaryClip(ClipData.newPlainText(null, text))
 }
 
-internal fun Context.createStreamThemeWrapper(): Context {
+internal fun Context.createStreamThemeWrapper(uiMode: UiMode = ChatUI.uiMode): Context {
     val typedValue = TypedValue()
+
     return when {
-        theme.resolveAttribute(R.attr.streamUiValidTheme, typedValue, true) -> this
-        theme.resolveAttribute(R.attr.streamUiTheme, typedValue, true) -> ContextThemeWrapper(this, typedValue.resourceId)
+        theme.resolveAttribute(R.attr.streamUiValidTheme, typedValue, true) ->
+            ContextThemeWrapper(this, R.style.StreamUiEmptyTheme)
```

Comment:
Is this line right?
If the `theme` into the context already has a valid theme, we override it by an empty theme??

Reply:
The original theme will only have the fields in the new theme overridden. As it is empty, all configuration stays the same.

Reply:
The next comment already deals with this comment.